server/routes/user.ts

```diff
@@ -11,10 +11,43 @@ import { UserType } from '../constants/user';
 
 const router = Router();
 
-router.get('/', async (_req, res) => {
+router.get('/', async (req, res) => {
   const userRepository = getRepository(User);
 
-  const users = await userRepository.find();
+  let sortFilter: FindOneOptions<MediaRequest>['order'] = {
+    id: 'ASC',
+  };
+
+  switch (req.query.sort) {
+    case 'updated':
+      sortFilter = {
+        updatedAt: 'DESC',
+      };
+      break;
+  }
+
+  let users = await userRepository.find({
+    order: sortFilter,
+  });
+
+  switch (req.query.sort) {
+    case 'requests':
+      users = users.sort(
+        (user1, user2) => user2.requestCount - user1.requestCount
+      );
```

Comment:
This query will have pagination in the future. This won't work. We need to build the query to do this.